### ISSUE TITLE
执行example.py train官方样例数据 加载完模型后无报错退出,未进入训练阶段

### ISSUE THREAD

**Issue by SunYusheng** [PaddlePaddle/RocketQA#72]
aistudio环境, A100 40G单卡
模型使用 rocketqa/zh_dureader_de_v2
训练数据使用examples/data/dual.train.tsv
即:
train_dual_encoder('zh_dureader_de_v2', './examples/data/dual.train.tsv')
执行到[     init.py:   74]:    Load pretraining parameters from /home/aistudio/.rocketqa/zh_dureader_de_v2/dual_params.
后退出
`
[INFO] 2022-11-29 10:26:59,291 [     args.py:   71]:    verbose: False
INFO:root:vocab_path: /home/aistudio/.rocketqa/zh_dureader_de_v2/zh_vocab.txt
[INFO] 2022-11-29 10:26:59,291 [     args.py:   71]:    vocab_path: /home/aistudio/.rocketqa/zh_dureader_de_v2/zh_vocab.txt
INFO:root:warmup_proportion: 0.1
[INFO] 2022-11-29 10:26:59,291 [     args.py:   71]:    warmup_proportion: 0.1
INFO:root:weight_decay: 0.01
[INFO] 2022-11-29 10:26:59,291 [     args.py:   71]:    weight_decay: 0.01
INFO:root:------------------------------------------------
[INFO] 2022-11-29 10:26:59,291 [     args.py:   72]:    ------------------------------------------------
INFO:root:Device count: 1
[INFO] 2022-11-29 10:26:59,330 [dual_encoder.py:  291]: Device count: 1
INFO:root:Num train examples: 112
[INFO] 2022-11-29 10:26:59,331 [dual_encoder.py:  292]: Num train examples: 112
INFO:root:Max train steps: 7
[INFO] 2022-11-29 10:26:59,331 [dual_encoder.py:  293]: Max train steps: 7
INFO:root:Num warmup steps: 0
[INFO] 2022-11-29 10:26:59,331 [dual_encoder.py:  294]: Num warmup steps: 0
INFO:root:Learning rate: 0.000010
[INFO] 2022-11-29 10:26:59,331 [dual_encoder.py:  295]: Learning rate: 0.000010
WARNING:root:paddle.fluid.layers.py_reader() may be deprecated in the near future. Please use paddle.fluid.io.DataLoader.from_generator() instead.
[WARNING] 2022-11-29 10:26:59,331 [       io.py:  721]: paddle.fluid.layers.py_reader() may be deprecated in the near future. Please use paddle.fluid.io.DataLoader.from_generator() instead.
INFO:rocketqa.utils.init:Load pretraining parameters from /home/aistudio/.rocketqa/zh_dureader_de_v2/dual_params.
[INFO] 2022-11-29 10:27:05,570 [     init.py:   74]:    Load pretraining parameters from /home/aistudio/.rocketqa/zh_dureader_de_v2/dual_params.
`

**Comment by yslu-TW** [comment]
#80 
兄弟 你遇到的問題跟我一樣，其實是有訓練的只是沒有儲存model跟列出log而已，要去他library的程式碼做修改，可參考我的回覆。
